cyber/py_wrapper/py_node.h

```diff
@@ -313,6 +338,98 @@ class PyNode {
   std::unique_ptr<Node> node_ = nullptr;
 };
 
+class PyChannelUtils {
+ public:
+  // Get debugstring of rawmsgdata
+  // Pls make sure the msg_type of rawmsg is matching
+  // Used in cyber_channel echo command
+  static std::string get_debugstring_by_msgtype_rawmsgdata(
+      const std::string& msg_type, const std::string& rawmsgdata) {
+    if (msg_type.empty()) {
+      AERROR << "parse rawmessage the msg_type is null";
+      return "";
```

Comment:
Maybe we can use `RETURN_IF_NULL` or `RETURN_IF` here. same situation with field `rawmsgdata`,  `channel_name ` and `raw_msg_class_` .